src/Compilers/CSharp/Portable/Symbols/Source/ParameterHelpers.cs

```diff
@@ -592,8 +603,19 @@ internal static void CheckParameterModifiers(BaseParameterSyntax parameter, Bind
                         }
                         break;
 
+                    case SyntaxKind.ReadOnlyKeyword:
```

Comment:
consider imitating what `scoped` and `params` are doing above by adding a `when !parsingFunctionPointerParams` here and leaving the `case SyntaxKind.ReadOnlyKeyword when parsingFunctionPointerParams` as-is.

Reply:
i see.  sure :)

Reply:
done.